marklogic-data-hub/src/test/java/com/marklogic/hub/hubcentral/AllArtifactsProject.java

```diff
@@ -45,35 +45,37 @@ public void writeProjectArtifactsToZipFile() {
         }
     }
 
+    // The path for the artifacts should not start "/" as it will not be picked up by Windows Winzip tool.
```

Comment:
Can you add one more assertion to this - iterate over the paths of all zip entries and verify that none of them start with a "/". That way, if/when we add another path in the future, we'll have a safeguard in place to ensure that it doesn't start with "/".